<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Init Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <property name="verticalSpacing">
       <number>21</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Initialization</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sources</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sourceNumber">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Servers</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="serverNumber">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Buffer Size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="bufferSize">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Source Intensity</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="sourceInt">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Min Service Time</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="serverIntMin">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Max Service Time</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="serverIntMax">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="completeBtn">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Complete Table</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="startBtn">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sourceNumber</tabstop>
  <tabstop>serverNumber</tabstop>
  <tabstop>bufferSize</tabstop>
  <tabstop>sourceInt</tabstop>
  <tabstop>serverIntMin</tabstop>
  <tabstop>serverIntMax</tabstop>
  <tabstop>startBtn</tabstop>
  <tabstop>completeBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
